addons/web_editor/static/src/js/wysiwyg/widgets/link.js

```diff
@@ -109,7 +109,7 @@ const Link = Widget.extend({
                 const subValues = value.split(',');
                 let subActive = true;
                 for (let subValue of subValues) {
-                    const classPrefix = new RegExp('(^|btn-| |btn-outline-)' + subValue);
+                    const classPrefix = new RegExp('(^|btn-| |btn-outline-|btn-fill-)' + subValue);
```

Comment:
Oh... the code we merged in 13.0 and 14.0 was actually already done by the editor team? Of course... why would they care about fixing stable versions at the same time :disappointed: